### ISSUE TITLE
[BUG] - Cannot do bulk delete because of too_long_frame_excepion

### ISSUE THREAD

**Issue by alucard001** [cars10/elasticvue#213]
**Description**
Assume that I have over 500 indices in elasticsearch (ES), and I want to delete it all in bulk.

If I check the "Select all" checkbox on the left of "Name" column, the result will show that I have selected all those 500 indices, even if I just show 10 indices per page.

And when I choose "Delete 5xx indices", it will show this exception:

```
{"error":{"root_cause":[{"type":"too_long_frame_exception","reason":"An HTTP line is larger than 4096 bytes."}],"type":"too_long_frame_exception","reason":"An HTTP line is larger than 4096 bytes."},"status":400}
```

**Environment (please include the following information):**
* Elasticsearch version: 6.5.4
* Operating system: Linux
* Browser + version: Chrome Version 121.0.6167.189 (Official Build) (64-bit)
* Elasticvue version: 1.0.3
* How are you running elasticvue? Browser extension

**Expected behavior:**
When I select "Select All", I would expect to select all items **on that page** only.  If you want to distinguish, you can have 2 options: "Select this page" / "Select everything" 

**Additional context**
Thanks for this amazing tool.

**Why do yo have 500+ indices?**
Just to let you know why I have 500+ indices: everyday I will create a backup of existing index, and create a brand new one, all by cronjob.  And then I will move the alias from original one to this new one.  Yet I leave it and did not touch this server for quite sometime.  And one day I remember this task and come back to delete those backup indices, this bug exists.

**Comment by cars10** [comment]
Bulk requests will be sent in chunks of 16 indices - exactly like kibana does it.

**Comment by alucard001** [comment]
But the indices are still there and cannot be removed.
![image](https://github.com/cars10/elasticvue/assets/1968244/4638de37-25b5-4d0b-bc2a-61582169abc9)

**Comment by cars10** [comment]
Sorry my answer was unclear. This was fixed in elasticvue `1.0.4` released yesterday.

**Comment by alucard001** [comment]
Oh。 I'm sorry I did not understand it well.  Thank you very much for your support.  Case can be closed.
